### ISSUE TITLE
[Component] - Tooltip

### ISSUE THREAD

**Issue by nickzafiropulos** [nearform/quantum#267]
Figma - https://www.figma.com/file/XFbhstkgQFz8ZefAU3w2p4/1.-Quantum-Design-System?type=design&node-id=88-5887&mode=design

Storybook - https://nearform.github.io/quantum/?path=/docs/components-tooltip--docs

1. Can we reduce the delay in the tooltip appearing when hovering on an element? It should be 0.1 seconds.
2. Can we leave the tooltip exposed for a little longer when the mouse leaves the element? It should be 0.5 seconds.
3. Please remove the 'different colour' variant. We should stick to 1 background colour only ( `accent/default` )

**Comment by nstolpe** [comment]
@nickzafiropulos I've opened a [PR](https://github.com/nearform/quantum/pull/276) that resolves 1 and 3. I couldn't find a way to configure the library we're using for the tooltips so that it would allow for a delay before it's hidden, but I did open a discussion on their GH page. If someone gets back to me with a solution we should be able to fix it.

I've  also added a [follow up issue](https://github.com/nearform/quantum/issues/277) so we can merge this and get 1 and 3 resolved.
